modular_skyrat/modules/verbs/code/looc_wallpierce.dm

```diff
@@ -0,0 +1,75 @@
+#define LOOC_RANGE 7
+
+/client/verb/looc_wallpierce(msg as text)
+	set name = "LOOC (Wall Piercing)"
+	set desc = "Local OOC, seen by anyone within 7 tiles of you."
+	set category = "OOC"
+
+	if(GLOB.say_disabled)	//This is here to try to identify lag problems
+		to_chat(usr, span_danger(" Speech is currently admin-disabled."))
+		return
+
+	if(!mob)
+		return
+
+	msg = copytext_char(sanitize(msg), 1, MAX_MESSAGE_LEN)
+	if(!msg)
+		return
+
+	if(!holder)
+		if(!GLOB.looc_allowed)
+			to_chat(src, span_danger(" LOOC is globally muted"))
+			return
+		if(handle_spam_prevention(msg,MUTE_OOC))
+			return
+		if(findtext(msg, "byond://"))
+			to_chat(src, span_boldannounce("<B>Advertising other servers is not allowed.</B>"))
+			log_admin("[key_name(src)] has attempted to advertise in LOOC: [msg]")
+			return
+		if(prefs.muted & MUTE_LOOC)
+			to_chat(src, span_danger("You cannot use LOOC (muted)."))
+			return
+		if(mob.stat)
+			to_chat(src, span_danger("You cannot use LOOC while unconscious or dead."))  //Skyrat change
```

Comment:
We're on Skyrat and this is modular, no need to indicate Skyrat changes lol

Reply:
I blame old looc code (resolved!)